src/app/screens/HomeTabs/Artists/ArtistTabs/ArtistDocuments/ArtistDocuments.tsx

```diff
@@ -0,0 +1,63 @@
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { TabsScrollView } from "app/wrappers"
+import { extractNodes } from "shared/utils"
+import { ArtistDocumentsQuery } from "__generated__/ArtistDocumentsQuery.graphql"
+import { ListEmptyComponent } from "app/sharedUI"
+import { useSpace } from "palette"
+import { GlobalStore } from "app/store/GlobalStore"
+import MasonryList from "@react-native-seoul/masonry-list"
+import { DocumentGridItem } from "app/sharedUI/items/DocumentGridItem"
+
+const demoLink =
+  "https://artsy-partner-engineering-documents-production.s3.amazonaws.com/DeopaIhC-KIDO4Rj-nZk5g/Design_aesthetics_principles_of_pleasure_in_design.pdf?X-Amz-Expires=43200&X-Amz-Date=20220714T132425Z&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAICYI665LIMIGJ6KQ/20220714/us-east-1/s3/aws4_request&X-Amz-SignedHeaders=host&X-Amz-Signature=c00a17a3112fd622fc00dd8f1b5ae33156306d16865971a10ff26364881743f3"
+
+export const ArtistDocuments = ({ slug }: { slug: string }) => {
+  const space = useSpace()
+  const selectedPartner = GlobalStore.useAppState((state) => state.activePartnerID)!
+  const artistDocumentsData = useLazyLoadQuery<ArtistDocumentsQuery>(artistDocumentsQuery, {
+    artistID: slug,
```

Comment:
```suggestion
   slug,
```
This could be just slug, just like on other queries